app/pathogen/arbovirus/analyze/columns.tsx

```diff
@@ -5,6 +5,7 @@ import React from "react";
 import { Button } from "@/components/ui/button";
 import { ArrowUpDown } from "lucide-react";
 import { pathogenColors } from "../dashboard/(map)/MapAndFilters";
+import validator from "validator";
```

Comment:
Commenting on random line for thread @HimanshuRanka 

> I think another enhancement on the Tech side I would love to see here is if we could pull the component one level up so it does not rerender between the analysis and the dashboard pages anymore since we do not need to send it filters. This would automatically make MapAndFilters redundant since now we would have a decoupled Map and Filters.
> I would assume moving it to the layout one level up should work.

Ouch actually NextJS makes this a bit more complicated than you might initially think it is. The real problem here is that I need it to be different between the pages and the layout doesn't help me at all here since it doesn't know what page it belongs. As far as decoupling Map and Filters even that doesn't come out as nice as you might initially think and I can't put it in the page because I need to pass props down to it now and NextJS doesn't like it if I do that. Maybe this is something we could discuss in person or maybe take a second look at later?

![image](https://github.com/PathoTracker/Pathotracker/assets/86806388/e6712f78-22cf-427f-bbba-059bb103bebb)